privacyidea/api/before_after.py

```diff
@@ -266,7 +266,13 @@ def after_request(response):
 @postrequest(sign_response, request=request)
 def auth_error(error):
     if "audit_object" in g:
-        g.audit_object.log({"info": error.message})
+        message = error.description
+
+        if hasattr(error, 'details'):
+            if 'message' in error.details:
+                message = "{}|{}".format(error.description, error.details['message'])
```

Comment:
This is a nice idea.
But please note, that e.g. ``AuthError`` has not ``description``.
See the failing tests at travis-ci. you can also run the tests in your virtualenv

    py.test tests/

So your code should check, if actually ``error`` has the attribute ``description``.